CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.52.1](https://github.com/brave/brave-ios/releases/tag/v1.52.1)
+
+- Resolved web compatibility issue on YouTube desktop site. ([#7714](https://github.com/brave/brave-ios/issues/7714))
+- Fixed a crash when reordering Favorites on New Tab Page. ([#7714](https://github.com/brave/brave-ios/issues/7714))
```

Comment:
small nit: usually, put quotes around `"Favorites" as it's easier to read and adds an emphasis on "Favorites".  Both issues are using the same issue # and link. Assuming copy/paste error.

Reply:
Thanks. Yes, the reason for using the same issue is that there was couple fixes done in 1.52.1 but only one issue in the milestone. There is context in the slack link in the issue. The fixes include web compatibility issue on YT and SDK changes to prevent some crashes.